### ISSUE TITLE
Some Training related questions

### ISSUE THREAD

**Issue by kunalverma75** [yangheng95/PyABSA#77]
1. Does the library have an early stopping mechanism so that the models don't overfit?
2. Can we specify a validation set/split during training?

**Comment by yangheng95** [comment]
Hello,
Both features are not available now, these features may be provided given the contributors have enough spare time.

**Comment by kunalverma75** [comment]
Ok thanks a lot
